pkg/services/user/userimpl/store.go

```diff
@@ -368,6 +368,9 @@ func (ss *sqlStore) ChangePassword(ctx context.Context, cmd *user.ChangeUserPass
 }
 
 func (ss *sqlStore) UpdateLastSeenAt(ctx context.Context, cmd *user.UpdateUserLastSeenAtCommand) error {
+	if cmd.UserID <= 0 {
+		return nil
```

Comment:
should we fail quietly or loudly here?

Reply:
in theory this should never be called with a zero userid, so loudly I guess

Reply:
```suggestion
		return user.ErrNoUniqueID
```

Reply:
I dislike reusing the same error in multiple places, but if the shoe fits...